### PR TITLE
FIxed issues

### PR DESCRIPTION
**Pull request by Vnarra558**

### CODE REVIEW

File: unredactor.tsv [hunk 1]
```diff
@@ -2703,95 +2703,95 @@ manishamandava01	testing	Wilson	██████ plays an obsessive sort, whic
 manishamandava01	testing	Ivana Milicervic	Another thing I loved was the strong character played by ████████████████.
 manishamandava01	testing	Porter	It's sad that anybody watching might actually think they were ██████'s own.
 Vnarra558	training	Matthew	Of course like many other people which bought this movie i saw ███████'s name, and that made me get it! and no his part isn't big at all, it's very short at the very end of the film, it's not a big part which makes me believe thats why people hate the film.
-Vnarra558	training	McConaughey	I don't understand why the other comments focus on ███████████.
+Vnarra558	training	McConaughey	I don't understand why the other comments focus on ███████████.
```
Comment (cegme) [inline unredactor.tsv:2706]: Trailing tabs

File: unredactor.tsv [hunk 2]
```diff
@@ -2703,95 +2703,95 @@ manishamandava01	testing	Wilson	██████ plays an obsessive sort, whic
...
 Vnarra558	training	Marian	Exquisite comedy starring ██████ Davies (with the affable William Haines).
 Vnarra558	training	Davies	Exquisite comedy starring Marian ██████ (with the affable William Haines).
-Vnarra558	training	Dorothy The appeal of the actors are what really put it over(John Ritter, Colleen Camp and the late ███████ Stratten are particularly good.) Go ahead and rent or buy this movie you'll be glad you did.
-Vnarra558	training	Bill Paxton	After all it's not everyday that someone comes in to your office to tell you that he knows who the killer is.The film has many twists, and ███████████ directs splendidly by keeping us guessing without losing interest.
+Vnarra558	training	Dorothy	The appeal of the actors are what really put it over(John Ritter, Colleen Camp and the late ███████ Stratten are particularly good.) Go ahead and rent or buy this movie you'll be glad you did.
+Vnarra558	training	Bill Paxton	After all it's not everyday that someone comes in to your office to tell you that he knows who the killer is.The film has many twists, and ███████████ directs splendidly by keeping us guessing without losing interest.
```
Comment (cegme) [inline unredactor.tsv:2713]: trailing tabs

**Review by cegme — CHANGES_REQUESTED** [review]
Thanks, very good. There are a few trailing tabs you can easily fix and resubmit. Thanks

File: unredactor.tsv (hunk 2 above)
Comment (Vnarra558) [reply unredactor.tsv:2713]: Ok I'll do it.

Comment (Vnarra558) [reply unredactor.tsv]: I fixed the trailing tabs.

**Review by cegme — CHANGES_REQUESTED** [review]
Please pull the latest and resubmit.

**Pull request closed without merging**